### PR TITLE
Random fact

### PR DESCRIPTION
**Pull request by jmithani**
Addresses Issue #9 

Adds two more facts to cycle through. Might need to lower the "Take the Challenge" button for aesthetic reasons, input appreciated.

cc: @jonuy

### CODE REVIEW

File: feeding-game/src/game/game.js [hunk 1]
```diff
@@ -328,7 +328,10 @@ function Game() {
     var pitch;
     var pitchButton;
     var pitchButtonText = "Give It A Try";
-    var pitchText = "94% of teens believe they would stay in school if they were pregnant. In reality, only 70% do. Think you can take care of a virtual baby for a day?";
+    //var pitchText = "94% of teens believe they would stay in school if they were pregnant. In reality, only 70% do. Think you can take care of a virtual baby for a day?";
+    var pitchChoose = ["94% of teens believe they would stay in school if they were pregnant. In reality, only 70% do. Think you can take care of a virtual baby for a day?", "51% of teens think if they were involved in a pregnancy they would marry the baby's mother or father. In reality, 81% remain unmarried. Think you could take care of a virtual baby by yourself for a day?","A sexually active teen who doesn’t use contraceptives has a 90% chance of becoming pregnant within a year. Think you can handle a virtual baby for a day?"];
```
Comment (jonuy) [inline feeding-game/src/game/game.js]: I'd break this out into one string per line for readability

```
var pitchChoose = [
  "94% of teens believe they would stay in school if they were pregnant. In reality, only 70% do. Think you can take care of a virtual baby for a day?", "51% of teens think if they were involved in a pregnancy they would marry the baby's mother or father. In reality, 81% remain unmarried. Think you could take care of a virtual baby by yourself for a day?",
  "A sexually active teen who doesn’t use contraceptives has a 90% chance of becoming pregnant within a year. Think you can handle a virtual baby for a day?"
];
```

File: feeding-game/src/game/game.js [hunk 2]
```diff
@@ -366,7 +369,7 @@ function Game() {
 
     // Button to go to campaign
     var px = game.world.centerX;
-    var py = pitch.position.y + margin * 6;
+    var py = pitch.position.y + margin * 6; //@todo possibly lower to make room for pitch text
```
Comment (jonuy) [inline feeding-game/src/game/game.js:374]: Yea, I'm down for lowering it if it's not getting in the way of other things below. Wherever it does get placed, it should look good whether the pitch text is 1, 2 or 3 lines large.

**Comment by jonuy** [comment]
Minor comments on readability above. Do you think there are more facts we can fill it in with?

Otherwise :+1:

**Comment by jmithani** [comment]
The facts were just ones I felt we could adequately make "grabby" enough to push people through to Pregnancy Text. It's probably more of a copy issue.

**Pull request merged**